### ISSUE TITLE
MQTT Discovery: Light objects should report `supported_color_modes`

### ISSUE THREAD

**Issue by dwmw2** [zwave-js/zwave-js-ui#3894]
### Checklist

- [x]  I am not using Home Assistant. Or: a developer has told me to come here.
- [x] I have checked the troubleshooting section and my problem is not described there.
- [x] I have read the changelog and my problem is not mentioned there.

### Deploy method

Manually built (git clone - npm run build - npm install

### Z-Wave JS UI version

9.18.1

### ZwaveJS version

13.3.0

### Describe the bug

Light objects are advertised for MQTT AutoDiscovery using an out-of-date method.

The documentation at https://developers.home-assistant.io/docs/core/entity/light/ says that *all* lights should report `supported_color_modes`, including those which only support `ColorMode.BRIGHTNESS` or `ColorMode.ONOFF`.

### To Reproduce

Use a Z-Wave light/dimmer device, and see the contents of its discovery topic.

### Expected behavior

I expected to see
```
  supported_color_modes: ["brightness"],
```
or
```
  supported_color_modes: ["onoff"],
```
in the device configuration object.

**Comment by robertsLando** [comment]
@dwmw2 Prepared a fix on #3895

**Comment by dwmw2** [comment]
That was fast. Thanks!
